libs/api-provider-environment/src/WebbProvider.tsx

```diff
@@ -488,22 +488,21 @@ export const WebbProvider: FC<WebbProviderProps> = ({ children, appEvent }) => {
           {
             let web3Provider: Web3Provider;
             if (wallet?.id === WalletId.WalletConnectV1) {
+              // Get rpcs from evm chains
+              const rpc = Object.values(chains).reduce((acc, chain) => {
+                if (chain.chainType === ChainType.EVM) {
+                  acc[chain.chainId] = chain.url;
+                }
+                return acc;
+              }, {} as Record<number, string>);
+
               const provider = new WalletConnectProvider({
                 rpc: {
+                  ...rpc,
+
                   //default on metamask
                   [EVMChainId.EthereumMainNet]:
                     'https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Ropsten]:
-                    'https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Goerli]:
-                    'https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Kovan]:
-                    'https://kovan.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
-                  [EVMChainId.Rinkeby]:
-                    'https://rinkeby.infura.io/v3/e54b7176271840f9ba62e842ff5d6db4',
-                  //default on metamask
-                  [EVMChainId.Beresheet]: 'http://beresheet1.edgewa.re:9933',
-                  [EVMChainId.HarmonyTestnet1]: 'https://api.s1.b.hmny.io',
```

Comment:
we can keep `Harmony1` and `Goerli`

Reply:
You have to make sure this endpoint is not deprecated first.